src/tests/ftest/util/server_utils_params.py

```diff
@@ -330,9 +330,7 @@ class PerEngineYamlParameters(YamlParameters):
             "common": [
                 "D_LOG_FILE_APPEND_PID=1",
                 "COVFILE=/tmp/test.cov"],
-            "ofi+sockets": [
-                "FI_SOCKETS_MAX_CONN_RETRY=5",
-                "FI_SOCKETS_CONN_TIMEOUT=2000",
+            "ofi+tcp": [
                 "CRT_SWIM_RPC_TIMEOUT=10"],
             "ofi_rxm": [
```

Comment:
Since "ofi+verbs" now means "ofi+verbs;ofi_rxm" and the provider will no longer include the `ofi_rxm` string, the `FI_OFI_RXM_USE_SRX=1` env will no longer be set by default.   To retain this behavior this should be updated to:
```suggestion
            "ofi+verbs": [
```

Reply:
I think with #7588 landed this should no longer be required. Ideally the test harness shouldn't be setting any of those environment variables...